package-lock.json

```diff
@@ -6466,6 +6561,11 @@
       "resolved": "https://registry.npmjs.org/etag/-/etag-1.8.1.tgz",
       "integrity": "sha1-Qa4u62XvpiJorr/qg6x9eSmbCIc="
     },
+    "eve": {
```

Comment:
are these new packages related to burger-menu or responsiveness?

Reply:
@harshvchawla  also the previous PR changed to fix conflits. I started working here before this one started.